pageserver/src/task_mgr.rs

```diff
@@ -135,6 +135,19 @@ pub static BACKGROUND_RUNTIME: Lazy<Runtime> = Lazy::new(|| {
         .expect("Failed to create background op runtime")
 });
 
+pub(crate) static RUNTIME_WORKER_THREADS: Lazy<usize> = Lazy::new(|| {
```

Comment:
Maybe this should be `BACKGROUND_RUNTIME_WORKER_THREADS` as I think there's some pressure to raise it up a multiple of 8. But it's very difficult to know how that'll affect, so I am proposing the limiting instead. Limit would still be useful even if we raised the number.

Reply:
[017cf24](https://github.com/neondatabase/neon/pull/4777/commits/017cf24cc04c44910811ec91b6a215acd3079e73)